utils/hct/hctbuild.cmd

```diff
@@ -52,6 +52,7 @@ set VENDOR=
 set SPIRV=OFF
 set SPV_TEST=OFF
 set DXILCONV=ON
+set DXC_CMAKE_SYSTEM_VERSION=10.0.14393
```

Comment:
This default define doesn't end in `.0`, which would match what was hard-coded before.  Perhaps this is the reason for the build failures in AppVeyor?